### ISSUE TITLE
thread 'main' panicked at 'not yet implemented: init command' on macos

### ISSUE THREAD

**Issue by seschis** [rojo-rbx/rojo#300]
### What did I do:
run 'rojo init' on a brandnew empty project dir

### What version
0.6.0-aplha.2

### What happend
➜  roblox-rodeo git:(master) RUST_BACKTRACE=full rojo init
thread 'main' panicked at 'not yet implemented: init command', /Users/shane/.cargo/registry/src/github.com-1ecc6299db9ec823/rojo-0.6.0-alpha.2/src/cli/init.rs:16:5
stack backtrace:
   0:        0x108c36655 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h253e83cc475a85d0
   1:        0x108c50091 - core::fmt::write::h5b2ae886983aa058
   2:        0x108c35f3b - std::io::Write::write_fmt::h25524112e4cce8f5
   3:        0x108c39b73 - std::panicking::default_hook::{{closure}}::h770880a3b7065f3d
   4:        0x108c3989f - std::panicking::default_hook::h8a113dafa31143b6
   5:        0x108c3a20b - std::panicking::rust_panic_with_hook::ha2f5dac743ef67b3
   6:        0x108c39da9 - rust_begin_unwind
   7:        0x108c733af - std::panicking::begin_panic_fmt::h360de49299ebe5b0
   8:        0x10869c907 - librojo::cli::init::init_inner::hac174b3240df024a
   9:        0x10869c880 - librojo::cli::init::init::h164a3bfb46d3f532
  10:        0x108649f7f - rojo::run::h25680ce0480ee955
  11:        0x10864b2d1 - std::panicking::try::do_call::h670f8e5205f80983
  12:        0x108c3bbef - __rust_maybe_catch_panic
  13:        0x108649983 - rojo::main::h216c8d67e5b5174b
  14:        0x108647cb6 - std::rt::lang_start::{{closure}}::h6491438ad5a20828
  15:        0x108c39c98 - std::panicking::try::do_call::hcc2f2b88c6645a8c
  16:        0x108c3bbef - __rust_maybe_catch_panic
  17:        0x108c2dcc5 - std::rt::lang_start_internal::h5c91a3e7c72ca35f
  18:        0x10864a4c9 - main
[ERROR rojo] Rojo crashed!
[ERROR rojo] This is a bug in Rojo.
[ERROR rojo]
[ERROR rojo] Please consider filing a bug: https://github.com/rojo-rbx/rojo/issues
[ERROR rojo]
[ERROR rojo] Details: not yet implemented: init command
➜  roblox-rodeo git:(master)

**Comment by Jaguar-515** [comment]
`rojo init` is not yet implemented as of now.

**Comment by LPGhatguy** [comment]
Jaguar is correct! They filed another issue about this too in #298 

We can keep one of these open as a reminder that this needs to happen before the 0.6.0 stable release!

**Comment by LPGhatguy** [comment]
Fixed in `master`!
